### DISCUSSION TITLE
PCL2主题更新RGB款式

### DISCUSSION THREAD

**Discussion by xiaoliziawa** [Meloong-Git/PCL#1124] [提问] [answered]
自定义主题是静态的，希望能动起来[doge]
让PCL的主题像RGB一样跑起来吧！

**Comment by WForst-Breeze** [comment, marked as answer]
Hmm... 现在 PCL2 已经有 **滑稽彩** 可以达到类似效果（不过很慢就是了），如果你想的话，可以前往提交 [新功能提案](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)

**Comment by LTDove** [comment]
> 自定义主题是静态的，希望能动起来[doge] 让PCL的主题像RGB一样跑起来吧！

但是那你可以去千万别点实现
